tests/longevity-large-partition-1TB-7days.yaml

```diff
@@ -0,0 +1,77 @@
+test_duration: 10080
+prepare_write_cmd:  ["scylla-bench -workload=uniform -mode=write -replication-factor=3 -partition-count=1000000 -clustering-row-count=1 -clustering-row-size=5120 -concurrency=10 -partition-offset=100 -rows-per-request=1 -duration=10080m",
```

Comment:
change the duration to be a bit less than 4 days, so the test_duration will be bigger (it's a timeout param).